Libraries/Components/View/ViewPropTypes.js

```diff
@@ -542,6 +542,13 @@ export type ViewProps = $ReadOnly<{|
    */
   'aria-hidden'?: ?boolean,
 
+  /**
+   * It reperesents the nativeID of the associated label text. When the assistive technology focuses on the component with this props, the text is read aloud.
+   *
+   * @platform android
+   */
+  'aria-labelledby'?: ?string | ?Array<string>,
```

Comment:
Array of string isn't the web type. It would be comma separated IDs in a single string